tests/regressions/FDB-310/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+ecbuild_add_executable( TARGET    fdb-url
```

Comment:
What does this tool do, and why is it stuck into a C++17 PR?

Reply:
Hm, the commit is not from me, something changed over the last 3 months. Think I just need to rebase on develop properly.